xbmc/pvr/addons/PVRClient.h

```diff
@@ -558,6 +559,23 @@ namespace PVR
      */
     time_t GetBufferTimeEnd() const;
 
+    /*!
+     * @return True if this add-on can be auto-configured via avahi, false otherwise
+     */
+    bool CanAutoconfigure(void) const;
+
+    /*!
+     * Registers the avahi type for this add-on
+     * @return True if registered, false if not.
+     */
+    bool AutoconfigureRegisterType(void);
```

Comment:
mwa, both auto-configure and autoconfigure are used in english afaict, this is better looking imo ;)